File: spring-cloud-square/square-provider/src/main/java/com/livk/provider/SquareProviderApp.java
package com.livk.provider;

import com.livk.commons.spring.SpringLauncher;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * <p>
 * SquareProviderApp
 * </p>
 *
 * @author livk
 * @date 2022/4/13
 */
@SpringBootApplication
public class SquareProviderApp {

    public static void main(String[] args) {
        SpringLauncher.run(SquareProviderApp.class, args);
    }

}

File: spring-cloud-square/square-provider/src/main/kotlin/com/livk/provider/SquareProviderApp.kt
package com.livk.provider

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

/**
 * @author livk
 */
@SpringBootApplication
open class SquareProviderApp

fun main(args: Array<String>) {
    runApplication<SquareProviderApp>(*args)
}
